libraries/Microsoft.Bot.Connector/Authentication/JwtTokenValidation.cs

```diff
@@ -84,20 +86,20 @@ public static async Task<ClaimsIdentity> ValidateAuthHeader(string authHeader, I
                 // No empty or null check. Empty can point to issues. Null checks only.
                 if (serviceUrl != null)
                 {
-                    return await ChannelValidation.AuthenticateChannelToken(authHeader, credentials, serviceUrl, httpClient ?? _httpClient, channelId);
+                    return await ChannelValidation.AuthenticateChannelToken(authHeader, credentials, serviceUrl, httpClient ?? _httpClient, channelId, authConfig);
                 }
```

Comment:
Null programming is ugly and bug prone here. 

Took a refactor in master today that cleaned this up:
[https://github.com/microsoft/botbuilder-dotnet/pull/1790](https://github.com/microsoft/botbuilder-dotnet/pull/1790)